src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -523,7 +523,7 @@ private String resolveContent(String result, Set<String> usedIds) {
      */
     public static Optional<String> getResolvedField(String field, BibEntry entry, BibDatabase database) {
         Objects.requireNonNull(entry, "entry cannot be null");
-        if ("bibtextype".equals(field)) {
+        if (BibEntry.TYPE_HEADER.equals(field) || "bibtextype".equals(field)) {
```

Comment:
I would guess that in the current master there will be different layout if you have entrytype and bintextype columns. We should probably add a migration that changes any bibtextype column to entrytype. This seems to have been neglected when changing the field name.